tests/routers/test_remote.py

```
@@ -0,0 +1,697 @@
+    TransferMode,
+    TrustStatus,
+    ColdStorageLocation
+)
+from app.schemas import RemoteConnectionCreate, RemoteTransferJob as RemoteTransferJobSchema
+
+# Assume authenticated_client, monitored_path_factory, storage_location fixtures are available.
+
+@pytest.fixture
+def remote_connection_factory(db_session: Session):
+    """Factory for RemoteConnection objects."""
+    def _factory(
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_factory`

(ANN202)